### PR TITLE
tkt-64392: Crash reporting opt-out

### PR DESCRIPTION
**Pull request by themylogin**
@william-gr I've made two assumptions while working on this, please agree or disagree:

1. Crash Reporting is undecided (which means off) for both existing and newly installed systems
2. In new UI we'll require to agree or disagree to enable crash reporting on first login. In legacy UI we don't want to spend our time on this, it'll be undecided until user opens System → General

### CODE REVIEW

File: gui/system/forms.py [hunk 1]
```diff
@@ -1029,6 +1030,9 @@ def middleware_clean(self, update):
         for key in ('ui_address', 'ui_v6address'):
             if not update.get(key):
                 update.pop(key, None)
+
+        update['crash_reporting'] = update['crash_reporting'] == 'True'
```
Comment (william-gr) [inline gui/system/forms.py]: Why True as string?

File: gui/system/migrations/0039_crash_reporting.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    dependencies = [
+        ('system', '0038_merge_20190226_1008'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='settings',
+            name='stg_crash_reporting',
+            field=models.NullBooleanField(help_text='Enable sending anonymous crash reports to iXsystems', verbose_name='Crash reporting'),
```
Comment (william-gr) [inline gui/system/migrations/0039_crash_reporting.py]: It should default to True for freenas, False for truenas.

File: gui/system/forms.py (hunk 1 above)
Comment (themylogin) [reply gui/system/forms.py]: `CheckboxInput` for `NullBooleanField` returns it this way for some reason

File: gui/system/migrations/0039_crash_reporting.py (hunk 2 above)
Comment (themylogin) [reply gui/system/migrations/0039_crash_reporting.py]: Fixed

**Pull request merged**